Kernel/Interrupts/APIC.cpp

```diff
@@ -62,6 +70,67 @@ namespace Kernel {
 
 static APIC *s_apic;
 
+class APICICIInterruptHandler final : public GenericInterruptHandler {
+public:
+    explicit APICICIInterruptHandler(u8 interrupt_vector):
+        GenericInterruptHandler(interrupt_vector, true)
```

Comment:
Please use `clang-format`

Reply:
We need to add a pre-commit hook with clang-format, I keep forgetting...